src/streamlink_cli/argparser.py

```diff
@@ -1545,6 +1554,7 @@ def build_parser():
     ("webbrowser_cdp_port", "webbrowser-cdp-port", None),
     ("webbrowser_cdp_timeout", "webbrowser-cdp-timeout", None),
     ("webbrowser_headless", "webbrowser-headless", None),
+    ("ffmpeg_validation_timeout", "ffmpeg-validation-timeout", None),
```

Comment:
`ffmpeg-validation-timeout` needs to be added to the actual Streamlink session options at `streamlink.session.options.StreamlinkOptions`.